constitution.tex

```diff
@@ -229,17 +235,17 @@ \subsection{Internal Interface} % Internal
 % Communicating with Working Group leaders
 % \todo{Membership management}
 
-\subsection{Student Body Interface} % Constituents
+\subsection{External Interface} % Constituents
 
-The Student Body Interface Working Group is responsible for maintaining communication with the rest of the SCS PhD student body. In particular, they are responsible for documentation practices, mass communication, onboarding, and processing Working Group proposals.
+The External Interface Working Group is responsible for maintaining communication with the rest of the SCS PhD student body. In particular, they are responsible for documentation practices, mass communication, onboarding, and processing Working Group proposals. 
 
-Although all Working Groups are responsible for producing documentation, the Student Body Interface Working Group is responsible for maintaining the central repository, sharing and maintaining best practices, and making sure that the documentation is accessible to committee members and the broader student body at all times. 
+Although all Working Groups are responsible for producing documentation, the External Interface Working Group is responsible for maintaining the central repository, sharing and maintaining best practices, and making sure that the documentation is accessible to committee members and the broader student body at all times. 
```

Comment:
I'm a little mixed on whether the internal or external should be responsible for documentation, guess it depends on how things end up being balanced